packages/dashboard/src/pages/CreateMaskWallet/components/CreateWalletForm/index.tsx

```diff
@@ -8,6 +8,10 @@ import { useNavigate } from 'react-router'
 import { RoutePaths } from '../../../../type'
 import { MaskColorVar } from '@masknet/theme'
 import { useDashboardI18N } from '../../../../locales'
+import { useAsyncRetry } from 'react-use'
+import { PluginServices } from '../../../../API'
+import { hasEncryptedWalletStore } from '../../../../../../maskbook/src/plugins/Wallet/database/decrypt'
```

Comment:
Use `currentIsMaskWalletLockedSettings`.

Reply:
I need to determine whether a crypto wallet currently exists instead of determining the lock status.